deck.gl/examples/pure-js/angular/firebase.json

```diff
@@ -0,0 +1,17 @@
+{
```

Comment:
If we're gonna keep the pub. info here, you might want to add a small description of the deployment process to readme, like we have in https://github.com/CartoDB/carto-react-template/blob/master/DEVELOPERS.md#deploying-the-sample-app

Reply:
Deployment instructions added